docs/tutorials/finetune.md

```diff
@@ -1,94 +1,209 @@
-# Tutorial 1: Finetuning Models
+# Tutorial 1: Fine-tune Models
 
-Classification models pre-trained on the ImageNet dataset has been demonstrated to be effective for other datasets and other downstream tasks.
-This tutorial provides instruction for users to use the models provided in the [Model Zoo](../model_zoo.md) for other datasets to obtain better performance.
+Classification models pre-trained on the ImageNet dataset have been demonstrated to be effective for other datasets and other downstream tasks.
+This tutorial provides instructions for users to use the models provided in the [Model Zoo](../model_zoo.md) for other datasets to obtain better performance.
 
-There are two steps to finetune a model on a new dataset.
+There are two steps to fine-tune a model on a new dataset.
 
 - Add support for the new dataset following [Tutorial 2: Adding New Dataset](new_dataset.md).
 - Modify the configs as will be discussed in this tutorial.
 
-Take the finetuning on CIFAR10 Dataset as an example, the users need to modify five parts in the config.
+Assume we have a ResNet-50 model pre-trained on the ImageNet-2012 dataset and want
+to take the fine-tuning on the CIFAR-10 dataset, we need to modify five parts in the
+config.
 
 ## Inherit base configs
 
-To reuse the common parts among different configs, we support inheriting configs from multiple existing configs. To finetune a ResNet-50 model, the new config needs to inherit
-`_base_/models/resnet50.py` to build the basic structure of the model. To use the CIFAR10 Dataset, the new config can also simply inherit `_base_/datasets/cifar10.py`. For runtime settings such as training schedules, the new config needs to inherit `_base_/default_runtime.py`.
+At first, create a new config file
+`configs/tutorial/resnet50_finetune_cifar.py` to store our configs. Of course,
+the path can be customized by yourself.
+
+To reuse the common parts among different configs, we support inheriting
+configs from multiple existing configs. To fine-tune a ResNet-50 model, the new
+config needs to inherit `configs/_base_/models/resnet50.py` to build the basic
+structure of the model. To use the CIFAR-10 dataset, the new config can also
+simply inherit `configs/_base_/datasets/cifar10_bs16.py`. For runtime settings such as
+training schedules, the new config needs to inherit
+`configs/_base_/default_runtime.py`.
+
+To inherit all above configs, put the following code at the config file.
 
 ```python
 _base_ = [
     '../_base_/models/resnet50.py',
-    '../_base_/datasets/cifar10.py', '../_base_/default_runtime.py'
+    '../_base_/datasets/cifar10_bs16.py', '../_base_/default_runtime.py'
 ]
 ```
 
-Besides, users can also choose to write the whole contents rather than use inheritance, e.g. `configs/mnist/lenet5.py`.
+Besides, you can also choose to write the whole contents rather than use inheritance, like `configs/mnist/lenet5.py`.
```

Comment:
it's better to add link to `configs/mnist/lenet5.py`.

Reply:
Done.